packages/jaeger-ui/vite.config.ts

```diff
@@ -83,7 +83,7 @@ export default defineConfig({
           '@layout-body-background': '#fff',
           '@layout-header-background': '#404040',
           '@layout-footer-background': '@layout-body-background',
-          '@layout-header-height': '64px',
+          '@layout-header-height': '48px',
```

Comment:
please add comments to all css changes to explain why they are necessary, so that we have a record. Otherwise it's very difficult to judge if the change is necessary or not.